README.md

```diff
@@ -1,11 +1,283 @@
-# OneMobileSDK Examples & Playgrounds
+# O2 Mobile SDK Tutorial – for iOS and tvOS <a name="head"></a>
 
-This repository contains a lot of examples and samples of OneMobileSDK usage.
+Welcome to the O2 Mobile SDK (OMSDK or SDK). The purpose of this document is to provide you with everything you need to know to get started with and use the O2 Mobile SDK. The audience for this document is an experienced iOS Developer with in-depth knowledge of using Xcode, the Swift Programming Language, iOS API programming, and either CocoaPods or Carthage.
 
-## Requirements:
-- Xcode 8/9
-- Swift 3.1/3.2
+This document will describe basic concepts and then will link you to sample projects and code, that are kept up-to-date with the latest versions of the the developer tools, API, and language we support. Our iOS and tvOS samples use CocoaPods as the default dependency manager (except for any Carthage-specific samples). 
 
-## Integration and usage guide:
+As always, we highly appreciate, welcome, and value all feedback on this documentation or the OMSDK in any way, shape, or form. If you have any suggestions for corrections, additions, or any further clarity, please don’t hesitate to email the [Video Support Team](mailto:video.support@oath.com).
 
-[Here](Usage%20guide.md)
+## Table of Contents
```

Comment:
This have to be autogenerated using http://doctoc.herokuapp.com

Reply:
ok - use it